tools/run-tests.py

```diff
@@ -503,6 +504,29 @@ def run_intl(engine, arch):
     if fails > 0:
         raise Exception('Intl tests failed')
 
+@runner('escargot-debugger', default=True)
+def run_escargot_debugger(engine, arch):
+    Escargot_Debugger_Test_DIR = join(PROJECT_SOURCE_DIR, 'tools', 'debugger', 'tests')
+    Escargot_Debugger_Client = join(PROJECT_SOURCE_DIR, 'tools', 'debugger', 'debugger.py')
+    Escargot_Debugger_Tester = join(PROJECT_SOURCE_DIR, 'tools', 'debugger', 'debugger_tester.sh')
```

Comment:
It's a minor suggestion. 
How about using uppercase for these variable names since all other variables are declared by uppercase names in this script file.